pallets/automation-time/src/lib.rs

```diff
@@ -177,6 +181,8 @@ pub mod pallet {
 		InvalidTime,
 		/// Time must be in the future.
 		PastTime,
+		/// Time cannot be too far in the future.
```

Comment:
do we need to explain why here for documentation purposes? and do we need to tell them exactly how far out in the UI?

Reply:
There is no real why. We just want a limit. How far out is based on environment. Will need to be in our API docs